types/src/connectors/api_handlers/create_connector.ts

```diff
@@ -1,36 +1,13 @@
 import * as t from "io-ts";
 
-export const CreateConnectorUrlRequestBodySchema = t.type({
-  url: t.string,
-  depth: t.number,
-  maxPages: t.number,
-  crawlMode: t.union([t.literal("child"), t.literal("website")]),
-  crawlFrequency: t.union([
-    t.literal("never"),
-    t.literal("daily"),
-    t.literal("weekly"),
-    t.literal("monthly"),
-  ]),
-});
-
-export const CreateConnectorOAuthRequestBodySchema = t.type({
-  connectionId: t.string,
-});
-
 export const ConnectorCreateRequestBodySchema = t.type({
   workspaceAPIKey: t.string,
   dataSourceName: t.string,
   workspaceId: t.string,
-  connectorParams: t.union([
-    CreateConnectorUrlRequestBodySchema,
-    CreateConnectorOAuthRequestBodySchema,
-  ]),
+  connectionId: t.string,
+  configuration: t.unknown,
```

Comment:
Can't we have `ConnectorConfiguration` here? or a schema that would be a disjunction of the configuration schema aka just `WebCrawlerConfigurationTypeSchema` here?